.github/workflows/pr-code-format.yml

```diff
@@ -31,12 +28,14 @@ jobs:
           separator: ","
           skip_initial_fetch: true
 
-      # We need to make sure that we aren't executing/using any code from the
-      # PR for security reasons as we're using pull_request_target. Checkout
-      # the target branch with the necessary files.
+      # We need to pull the script from the main branch, so that we ensure
+      # we get a version of the script that supports the --write-comment-to-file
```

Comment:
Nit: I think we should probably reword this to say something about making sure the script is updated rather than specifically mentioning the `--write-comment-to-file` option, which I think would quickly become a dated comment.